tests/models/documents/test_document_verbs.py

```diff
@@ -226,10 +222,16 @@ def test_force_reparse_empty(self, sns, mock_api_client):
     @patch.dict(os.environ, {"PRIVATE_ASSET_BUCKET": "MY_BUCKET"})
     @patch.dict(os.environ, {"REPARSE_SNS_TOPIC": "MY_TOPIC"})
     def test_force_reparse_full(self, sns, mock_api_client):
-        document = JudgmentFactory().build(is_published=True)
-        document.api_client = mock_api_client
+        document = Judgment("test/2023/123", mock_api_client)
+
+        document.neutral_citation = "[2023] Test 123"
+        document.consignment_reference = "TDR-12345"
+
+        document.body.name = "Judgment v Judgement"
+        document.body.court = "Court of Testing"
+        document.body.document_date_as_date = datetime.date(2023, 2, 3)
 
-        Judgment.force_reparse(document)
+        document.force_reparse()
```

Comment:
Ah; I think the reason I was doing it the other way was that the mock document wasn't actually a judgment. Having an actual document is much cleaner.